spec/features/state_file/complete_intake_spec.rb

```diff
@@ -90,38 +90,48 @@
       select_cfa_date "state_file_ny_spouse_state_id_form_expiration_date", Time.now + 4.year
       select("New York", from: I18n.t('state_file.questions.ny_primary_state_id.state_id.id_details.issue_state'))
       fill_in  I18n.t('state_file.questions.ny_primary_state_id.state_id.id_details.first_three_doc_num'), with: "ABC"
-      click_on "Continue"
+      click_on I18n.t("general.continue")
 
       expect(page).to have_text "The page with all the info from the 201"
-      click_on "Continue"
+      click_on I18n.t("general.continue")
 
       expect(page).to have_text "The page with all the info from the IT-214"
-      click_on "Continue"
+      click_on I18n.t("general.continue")
 
       expect(page).to have_text I18n.t('state_file.questions.unemployment.edit.title')
-      choose "Yes"
+      choose I18n.t("general.affirmative")
       choose "NYS Department of Labor"
       choose I18n.t('state_file.questions.unemployment.edit.recipient_myself')
       choose I18n.t('state_file.questions.unemployment.edit.confirm_address_yes')
       fill_in I18n.t('state_file.questions.unemployment.edit.unemployment_compensation'), with: "123"
       fill_in I18n.t('state_file.questions.unemployment.edit.federal_income_tax_withheld'), with: "456"
       fill_in I18n.t('state_file.questions.unemployment.edit.state_income_tax_withheld'), with: "789"
-      click_on "Continue"
+      click_on I18n.t("general.continue")
 
       expect(page).to have_text(I18n.t('state_file.questions.unemployment.index.1099_label', name: StateFileNyIntake.last.primary.full_name))
-      click_on "Continue"
+      click_on I18n.t("general.continue")
+
+      # From the review page, the user can go back to certain screens to edit and then should return directly to the
+      # review page. This is well-covered by unit tests, but let's test just one of those screens here
+      expect(page).to have_text I18n.t("state_file.questions.ny_review.edit.title1")
+      within "#county" do
+        click_on I18n.t("general.edit")
+      end
+      expect(page).to have_text I18n.t("state_file.questions.ny_county.edit.title", filing_year: MultiTenantService.statefile.current_tax_year)
+      click_on I18n.t("general.continue")
+      expect(page).to have_text I18n.t("state_file.questions.ny_review.edit.title1")
```

Comment:
i just kept one screen in each state flow, now that everything is covered in controller unit tests too